code/game/machinery/doors/airlock.dm

```diff
@@ -1037,14 +1037,12 @@
 	else if(locked)
 		to_chat(user, "<span class='warning'>The airlock's bolts prevent it from being forced!</span>")
 	else if( !welded && !operating)
-		if(!beingcrowbarred) //being fireaxe'd
+		if(istype(I, /obj/item/twohanded/fireaxe)) //being fireaxe'd (maybe)
```

Comment:
Don't you still need the `beingcrowbarred` check?

Reply:
The `beingcrowbarred` check was only being used to decide whether to check for a fireaxe`.